modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,290 @@
+            "ingestionTime": 1629422744016
+        },
+...
+----
+
+.Example: Replacing the default `infrastructureName` prefix in log group names
+
+You can replace the default `infrastructureName` prefix, `myusername-48-7977k`, in the log group names by specifying a `groupPrefix` field in the `ClusterLogForwarding` CR:
+
+[source,yaml]
+----
+cloudwatch:
+    region: us-east-2
+    groupBy: logType
+    groupPrefix: demo-group-prefix
+----
+
+The value of `groupPrefix` replaces the default `infrastructureName` prefix:
+
+[source,terminal]
+----
+$ aws --profile osd-logging-team logs describe-log-groups | jq .logGroups[].logGroupName
+"demo-group-prefix.application"
+"demo-group-prefix.audit"
+"demo-group-prefix.infrastructure"
+----
+
+.Example: Creating log groups for application namespace names
+
+You can create separate log groups for each application namespace name by setting the value of the `groupBy` field to `namespaceName` in the `ClusterLogForwarder` CR.
+
+[source,terminal]
+----
+cloudwatch:
+    region: us-east-2
+    groupBy: namespaceName
+----
+
+[NOTE]
+====
+This configuration does not create log groups for non-application namespaces, such as `openshift*`
+====
+
+In AWS, the namespace name appears at the end of each log group name. Because there is a single application namespace, "app", the following output shows a new `myusername-48-7977k.app` log group instead of `myusername-48-7977k.application`:
+
+[source,terminal]
+----
+$ aws --profile osd-logging-team logs describe-log-groups | jq .logGroups[].logGroupName
+"myusername-48-7977k.app"
+"myusername-48-7977k.audit"
+"myusername-48-7977k.infrastructure"
+----
+
+If the cluster in this example had contained multiple application namespaces, the output would show multiple log groups, one for each namespace.
+
+The `groupBy` field affects the application log group only. It does not affect the `audit` and `infrastructure` log groups.
+
+.Example: Creating log groups for application namespace UUIDs
+
+You can create separate log groups for each application namespace UUID by setting the value of the `groupBy` field to `namespaceUUID` in the `ClusterLogForwarder` CR.
+
```

Comment:
Destroying then creating a namespace object with the same name results in a new log group. Use when it is important to distinguish logs from successive namespaces instances with the same name.

Reply:
Updated. Thanks.